bumble/gatt_client.py

```diff
@@ -216,8 +232,10 @@ class ProfileServiceProxy:
     Base class for profile-specific service proxies
     '''
 
+    SERVICE_CLASS: type[TemplateService]
```

Comment:
using lowercase `type[]` is only for Python >= 3.9. For Python 3.8, we still need to use `Type[]`.